examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -1,100 +1,141 @@
+# scikit-learn, and print its description:
+from sklearn.datasets import load_diabetes
 
 diabetes = load_diabetes()
+X, y = diabetes.data, diabetes.target
+print(diabetes.DESCR)
 
-X = diabetes.data
-y = diabetes.target
+# %%
+# Feature importance from coefficients
+# ------------------------------------
+#
+# To get an idea of the importance of the features, we are going to use the
+# :class:`~sklearn.linear_model.LassoCV` estimator. The features with the
+# highest absolute `coef_` value are considered the most important.
+# We can observe the coefficients directly without needing to scale them (or
+# scale the data) because from the description above, we know that the features
+# were already standardized.
+# For a more complete example on the interpretations of the coefficients of
+# linear models, you may refer to
+# :ref:`sphx_glr_auto_examples_inspection_plot_linear_model_coefficient_interpretation.py`.
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.linear_model import LassoCV
 
-feature_names = diabetes.feature_names
-print(feature_names)
+lasso = LassoCV().fit(X, y)
+importance = np.abs(lasso.coef_)
+feature_names = np.array(diabetes.feature_names)
+plt.bar(height=importance, x=feature_names)
+plt.title("Feature importances via coefficients")
+plt.show()
 
 # %%
-# Find importance of the features
-# ---------------------------------------------------------
+# Selecting features based on importance
+# --------------------------------------
 #
-# To decide on the importance of the features we are going to use LassoCV
-# estimator. The features with the highest absolute `coef_` value are
-# considered the most important
+# Now we want to select the two features which are the most important according
+# to the coefficients. The :class:`~sklearn.feature_selection.SelectFromModel`
+# is meant just for that. :class:`~sklearn.feature_selection.SelectFromModel`
+# accepts a `threshold` parameter and will select the features whose importance
+# (defined by the coefficients) are above this threshold.
+#
+# Since we want to select only 2 features, we will set this threshold slightly
+# above the coefficient of third most important feature.
+from sklearn.feature_selection import SelectFromModel
+from time import time
 
-clf = LassoCV().fit(X, y)
-importance = np.abs(clf.coef_)
-print(importance)
+threshold = np.sort(importance)[-3] + 0.01
+
+tic = time()
+sfm = SelectFromModel(lasso, threshold=threshold).fit(X, y)
+toc = time()
+print("Features selected by SelectFromModel: "
+      f"{feature_names[sfm.get_support()]}")
+print(f"Done in {toc - tic:.3f}s")
 
 # %%
-# Select from the model features with the higest score
-# ---------------------------------------------------------
+# Selecting features with Sequential Feature Selection
+# ----------------------------------------------------
+#
+# Another way of selecting features is to use Sequential Feature Selection
```

Comment:
It would be nice to have a link to the documentation directly here

```suggestion
# Another way of selecting features is to use
:class:`~sklearn.feature_selection.SequentialFeatureSelector`
```